content/automate/ManageIQ/System/Event/EmsEvent/NUAGE.class/_missing.yaml

```diff
@@ -0,0 +1,12 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: 
+    name: ".missing"
+    inherits: 
+    description: 
+  fields:
+  - rel4:
+      value: "/System/event_handlers/refresh"
```

Comment:
so before @gmcculloug and @blomquisg were rather for manually defining handlers for every event, instead of method_missing. Since the handlers also solve as documentation of 'these are the events we support'